lib/gui/models/settings.js

```diff
@@ -20,12 +20,15 @@
  * @module Etcher.Models.Settings
  */
 
-const angular = require('angular');
 const Store = require('./store');
-const MODULE_NAME = 'Etcher.Models.Settings';
-const SettingsModel = angular.module(MODULE_NAME, []);
 
-SettingsModel.service('SettingsModel', function() {
+/**
+ * @summary Setting/getting values
+ * @class
+ * @public
+ *
+ */
+class SettingsModel {
```

Comment:
This class is only exposing static functions, so I think we should simply attach these functions to `exports`, like in https://github.com/resin-io/etcher/pull/1251/files